net-misc/uftp/uftp-5.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Encrypted UDP based FTP with multicast"
+HOMEPAGE="http://uftp-multicast.sourceforge.net/"
+SRC_URI="https://download.sourceforge.net/${PN}-multicast/source-tar/${P}.tar.gz"
+
+LICENSE="GPL-3-with-openssl-exception"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+server ssl"
+
+# openssl with EC required, see #644674
+DEPEND="ssl? ( dev-libs/openssl:0=[-bindist] )"
+RDEPEND="${DEPEND}"
+
+# Workaround, see #644670
+RESTRICT=test
```

Comment:
Still needed?

Reply:
Well, I have same concerns. Code don't have any test framework anymore, but I decided to left this restrict as safecheck.